code/modules/paperwork/paper_bundle.dm

```diff
@@ -0,0 +1,250 @@
+/obj/item/paper_bundle
+	name = "paper bundle"
+	gender = PLURAL
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "paper"
+	item_state = "paper"
+	throwforce = 0
+	w_class = 1.0
+	throw_range = 2
+	throw_speed = 1
+	layer = 4
+	pressure_resistance = 1
+	attack_verb = list("bapped")
+	var/amount = 0 //Amount of items clipped to the paper
+	var/page = 1
+	var/screen = 0
+	/// If this was sent via admin fax, allows anyone to see/interact with it
+	var/admin_faxed = FALSE
+
+
+/obj/item/paper_bundle/attackby(obj/item/W as obj, mob/user as mob)
```

Comment:
do we use this typing elsewhere in our code? Only other place I see it is modular computer

Reply:
oops